src/Log/Migrations/MigrateExistingLogs.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Give\Log\Migrations;
+
+use Give\Log\LogType;
+use Give\Log\LogFactory;
+use Give\Log\LogCategory;
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+
+class MigrateExistingLogs extends Migration {
```

Comment:
Since we're handling this in batches, we will need to make this an update. Some existing sites may very well have thousands of migrations, and this would max out memory. See the example Ravinder pointed to. If you have any ideas of how we can improve this, please share.

Reply:
Resolved: 884a1836

Reply:
I have an idea how to improve this. I'll write a pitch for it.